instance/models/load_balancer.py

```diff
@@ -183,24 +193,28 @@ def run_playbook(self, ansible_vars):
         This is factored out into a separate method so it can be mocked out in the tests.
         """
         playbook_path = pathlib.Path(settings.SITE_ROOT) / "playbooks/load_balancer_conf/load_balancer_conf.yml"
-        returncode = ansible.capture_playbook_output(
-            requirements_path=str(playbook_path.parent / "requirements.txt"),
-            inventory_str=self.domain,
-            vars_str=ansible_vars,
-            playbook_path=str(playbook_path),
-            username=self.ssh_username,
-            logger_=self.logger,
-        )
+        with cache.lock(self.domain):
```

Comment:
nit: maybe use a more specific lock name, such as `"{}-configure-load-balancer".format(self.domain)`.

Do we know what happens if the process crashes while holding the lock? Does the lock ever get released?